climakitae/selectors.py

```diff
@@ -543,45 +514,43 @@ def view(self):
             scatter_size = 1.5
 
         # Add grid boundaries
-        if "Statistical" in self.selections.downscaling_method:
-            # 3km LOCA grid shown whenever LOCA is selected, even if WRF is also selected
+        # if "Statistical" in self.selections.downscaling_method:
+        #     # 3km LOCA grid shown whenever LOCA is selected, even if WRF is also selected
+        #     _add_res_to_ax(
+        #         poly=self._loca_bb["3 km"],
+        #         ax=ax,
+        #         color="purple",
+        #         rotation=32,
+        #         xy=(-127, 40),
+        #         label="3 km",
+        #     )
```

Comment:
Better not to merge commented-out work-in-progress code. Stash it elsewhere.

Reply:
Good catch. Removed!